x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -1974,4 +1996,235 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'windows.advanced.events.aggregate_process',
+    first_supported_version: '8.16',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.windows.advanced.events.aggregate_process',
+      {
+        defaultMessage:
+          'Reduce event volume by merging related <network or process> events into fewer aggregate events. Default is true.',
```

Comment:
It's a bit weird to have `aggregate_process` aggregate both process and network events.  Should we rename it or add a separate toggle for network events?  @nicholasberlin

Reply:
The original plan was to add network aggregation as a separate toggle, but I wasn't able to complete the work. So, we will be adding a new toggle `aggregate_network` for 8.17.

This wording is my fault, because of lack of clarity. I meant for either `network` or `process` to be chosen based on the toggle name.
```suggestion
          'Reduce event volume by merging related process events into fewer aggregate events. Default is true.',
```

Reply:
yes, sorry, i just noticed it and fixed it some hours ago, just haven't pushed it :D thank you!
5b967095da666e027ced5311bb3a8077efba9c8f